### PR TITLE
feat:  #55348 Move Serach into left panel

### PR DESCRIPTION
**Pull request by ingridklepsvik**
### Description

Search icon and exit icon are moved into the main left panel

Before 
image

After: 
image

### CODE REVIEW

File: packages/EchoFramework/src/components/panel/corePanelLeft.tsx [hunk 1]
```diff
@@ -19,7 +19,12 @@ const CorePanelLeft: React.FC = () => {
 
     return (
         <>
-            <div className={`${style.wrapper} ${isPanelActive ? style.active : ''}`} style={panelUI.panelWrapper}>
+            <div
+                className={`${isPanelActive ? style.activeWrapper : style.wrapper} ${
```
Comment (jorgenholme96) [inline packages/EchoFramework/src/components/panel/corePanelLeft.tsx:23]: Kan `active` og `activeWrapper` klassene kombineres? Ser vi har samme sjekk på de

Comment (ingridklepsvik) [reply packages/EchoFramework/src/components/panel/corePanelLeft.tsx]: Ja, de kan kombineres. Skal være fikset nå

**Review by jorgenholme96 — APPROVED** [review]

File: packages/EchoFramework/src/components/panel/corePanelLeft.module.css [hunk 2]
```diff
@@ -9,12 +9,27 @@
     transition: all 0.4s ease-in-out;
 }
 
+.activeWrapper {
+    position: fixed;
+    z-index: 2;
```
Comment (hkot1991) [inline packages/EchoFramework/src/components/panel/corePanelLeft.module.css:14]: Isnt this a very low z-index?

Comment (hkot1991) [reply packages/EchoFramework/src/components/panel/corePanelLeft.module.css:14]: I believe the z-index for context menu is 2 aswell

Comment (hkot1991) [reply packages/EchoFramework/src/components/panel/corePanelLeft.module.css:14]: Maybe this is not a problem? Is i tested that pdf context menu dont overlap on this?

Comment (hkot1991) [reply packages/EchoFramework/src/components/panel/corePanelLeft.module.css:14]: isnt this the same styling as the .wrapper ? Why do you need to the same styling multiple times?

Feel like Im having a rant with my self here, sorry about this 😅

File: packages/EchoFramework/src/components/panel/corePanelLeft.module.css [hunk 3]
```diff
@@ -9,12 +9,27 @@
...
+}
+
 @media screen and (max-width: 550px) {
     .wrapper {
         left: -75vw;
         width: 75vw;
         max-width: 75vw;
     }
+
+    .activeWrapper {
```
Comment (hkot1991) [inline packages/EchoFramework/src/components/panel/corePanelLeft.module.css:30]: overkill to add activewrapper? 
just add width: 100% to active class and undo the change in component should give the same result?

File: packages/EchoFramework/src/components/panel/corePanelLeft.module.css [hunk 4]
```diff
@@ -9,12 +9,27 @@
     transition: all 0.4s ease-in-out;
 }
 
+.activeWrapper {
```
Comment (hkot1991) [inline packages/EchoFramework/src/components/panel/corePanelLeft.module.css:12]: this is the same as .wrapper class, so suggest see the other comment I added or:
.activeWrapper, .wrapper {....these have the same style....}

**Review by hkot1991 — APPROVED** [review]
🚀
